app/views/submissions/_form.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= simple_form_for @submission  do |form| %>
+  <% if @submission.errors %>
+    <%= render "errors", object: @submission %>
+  <% end %>
+
+  <%= form.input :title %>
+  <%= form.input :submitted_to %>
+  <%= form.select :status, Submission::SUBMISSION_STATUSES %>
+  <%= form.association :poems, as: :check_boxes %>
+  <%= form.input :user_id, as: :hidden, input_html: { value: current_user.id } %>
```

Comment:
I tend to put these kinds of variables into the controller itself, so a user can't mess with the html and assign it to something they shouldn't. Because `user_id` is a permitted param, they could edit it to be any user in the system.

A pattern I've done a few times now that I like is to have a different controller method for the create action, that injects the current user specific things. Something like

```ruby
def new_submission_params
  submission_params.merge(user_id: current_user.id)
end

def submission_params
  params.require(:submission).
  permit(:title, :submitted_to, :status, poem_ids: [])
end
```

Where the create action would use `new_submission_params` and the update action would use `submission_params`.